### PR TITLE
Add toggle function, started gradually working on backend, completed …

### PR DESCRIPTION
**Pull request by L-Kunu**
…stories for Button.js and SearchPage.js. In process/ stories for Card.js, but there's an error message that one of my props is undefined

### CODE REVIEW

**Review by CasparVelsen — APPROVED** [review]
sieht super aus

File: src/App.js [hunk 1]
```diff
@@ -4,16 +4,18 @@ import styled from "styled-components";
 import { nanoid } from "nanoid";
 import SearchPage from "./components/SearchPage.js";
 import { useState } from "react";
+import useLocalStorage from "./Hooks/UseLocalStorage.js";
```
Comment (Sven-Lyco) [inline src/App.js:7]: You should rename your file :) 
```suggestion
import useLocalStorage from "./Hooks/useLocalStorage.js";
```

File: src/App.js [hunk 2]
```diff
@@ -4,16 +4,18 @@ import styled from "styled-components";
...
   const id = nanoid();
   const [immigrationSearch, setImmigrationSearch] = useState(false);
   const [familySearch, setFamilySearch] = useState(false);
-  const [LawyerList, setLawyerList] = useState(Lawyers);
+  const [LawyerList, setLawyerList] = useLocalStorage("LawyerList", false);
 
   const immigrationLaw = Lawyers.filter((lawyer) =>
     lawyer.specialty.includes("Immigration law")
   );
+  console.log(immigrationLaw);
```
Comment (Sven-Lyco) [inline src/App.js:18]:
```suggestion

```

File: src/components/SearchPage.js [hunk 3]
```diff
@@ -35,22 +34,27 @@ export default function SearchPage({ onSubmit }) {
...
 }
 
 const Form = styled.form`
+background-color: #A7BBC7;
+border-radius: 14px;
   display: flex;
   flex-direction: column;
   justify-content: center;
+  padding: 16px;
+  //opacity: 0.8;
```
Comment (Sven-Lyco) [inline src/components/SearchPage.js:56]:
```suggestion

```

**Review by Sven-Lyco — COMMENTED** [review]
Looks good to me, just a few comments :)

**Pull request merged**